staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/status.go

```diff
@@ -59,10 +59,12 @@ func ErrorToAPIStatus(err error) *metav1.Status {
 		return &status
 	default:
 		status := http.StatusInternalServerError
+		reason := metav1.StatusReasonUnknown
 		switch {
 		//TODO: replace me with NewConflictErr
 		case storage.IsConflict(err):
 			status = http.StatusConflict
+			reason = metav1.StatusReasonConflict
```

Comment:
This looks reasonable, but is this code actually reachable? IsConflict only returns true for storage errors with code == ErrCodeResourceVersionConflicts. That appears to only be constructed in NewResourceVersionConflictsError, and there are no callers to that function.

Are you actually seeing this response from a real server?